modules/openapi-generator/src/main/java/org/openapitools/codegen/DefaultCodegen.java

```diff
@@ -3264,13 +3264,16 @@ protected void addAdditionPropertiesToCodeGenModel(CodegenModel codegenModel, Sc
      * @param required   required property only
      * @param schema     schema in which the properties will be added to the lists
      */
-    protected void addProperties(Map<String, Schema> properties, List<String> required, Schema schema) {
```

Comment:
@kthoms thanks for the PR. Please update the docstring of the method as well since it's causing issues to the build: https://github.com/OpenAPITools/openapi-generator/runs/5213864795?check_suite_focus=true

Reply:
I have updated the code and rebased it. The build fails in the samples. I don't think that it is related, WDYT?